docs/_blog/bdk_cli_basics_00.md

```diff
@@ -0,0 +1,294 @@
+
+![](https://i.imgur.com/EUCovcJ.gif)
+
+
+
+:+1: The output below confirms the command was successful.
+
+```
+{
+  "details": {
+    "confirmation_time": null,
+    "fee": 113,
+    "received": 0,
+    "sent": 123000,
+    "transaction": null,
+    "txid": "029173d76253e3441f9dc26f91e6ef30dff486848e91a7941f0cacd0af25ee30"
+  },
+  "psbt": "cHNidP8BAFUBAAAAAak8uMR3UGkAGUKWsq8Mv45qg2fdD93JQRIsa2P0wFloAQAAAAD/////AQfgAQAAAAAAGXapFDRKD0jKFQ7CuQOBdmC5tosTpnAmiKwAAAAAAAEA3gIAAAAAAQFY9sVfEEbyjrHXSlxXDxL+71WOMnsPpVElwk+3E/J9vAAAAAAA/v///wIYZRIAAAAAABYAFBKYf7yF+ss6EFdw2rDZTfdLhep8eOABAAAAAAAWABQd6wBRnX8mBZiJYfvy/FGE+xKc+AJHMEQCIFSIkvEUI9yUgEw4JocRs1aiVsBlKKXrOQaQb3XFqR21AiBqiEVzCVVSRGjckyPDgAQBnOdSzBYR6Rw6KFcCP+E27wEhAwIlXdfM2WYnYa36Hp4MS6YkplBAgBsb1tYG9NiWFWTKzPYhAAEBH3jgAQAAAAAAFgAUHesAUZ1/JgWYiWH78vxRhPsSnPgiBgP80FpaWYQzGzCnNI9blXbei61YpAmtoezMRxpVvBJ6SxgTizKsVAAAgAEAAIAAAACAAAAAAAAAAAAAAA=="
+}
+
+```
+ 
+
+
+***
+### 6a: export PSBT to environment-variable
+ :arrow_forward: `export PSBT="PASTE_PSBT_HERE"`
+
+![](https://i.imgur.com/CEDKcPZ.gif)
+
+
+ 
+
+***
+## Step 7: Sign Transaction (PSBT) 
+
+:arrow_forward: ` bdk-cli wallet --wallet wallet_name --descriptor $my_descriptor sign --psbt $PSBT`
+- DON'T FORGET to COPY the PSBT for the next step
+
+![](https://i.imgur.com/f4o4Ce8.gif)
+
+
+:+1: The output below confirms the command was successful.
+
+
+
+```
+{
+  "is_finalized": true,
+  "psbt": "cHNidP8BAFUBAAAAAak8uMR3UGkAGUKWsq8Mv45qg2fdD93JQRIsa2P0wFloAQAAAAD/////AQfgAQAAAAAAGXapFDRKD0jKFQ7CuQOBdmC5tosTpnAmiKwAAAAAAAEA3gIAAAAAAQFY9sVfEEbyjrHXSlxXDxL+71WOMnsPpVElwk+3E/J9vAAAAAAA/v///wIYZRIAAAAAABYAFBKYf7yF+ss6EFdw2rDZTfdLhep8eOABAAAAAAAWABQd6wBRnX8mBZiJYfvy/FGE+xKc+AJHMEQCIFSIkvEUI9yUgEw4JocRs1aiVsBlKKXrOQaQb3XFqR21AiBqiEVzCVVSRGjckyPDgAQBnOdSzBYR6Rw6KFcCP+E27wEhAwIlXdfM2WYnYa36Hp4MS6YkplBAgBsb1tYG9NiWFWTKzPYhAAEBH3jgAQAAAAAAFgAUHesAUZ1/JgWYiWH78vxRhPsSnPgiAgP80FpaWYQzGzCnNI9blXbei61YpAmtoezMRxpVvBJ6S0gwRQIhALWkBRSJzxuf0od4tPu3qFmEfJ2Y+/QBGtfjSFObWsPeAiA4QJx8Rk5pacrjHv5EOdw6RNHXcdtepFs+m0/Za/h0UQEiBgP80FpaWYQzGzCnNI9blXbei61YpAmtoezMRxpVvBJ6SxgTizKsVAAAgAEAAIAAAACAAAAAAAAAAAABBwABCGwCSDBFAiEAtaQFFInPG5/Sh3i0+7eoWYR8nZj79AEa1+NIU5taw94CIDhAnHxGTmlpyuMe/kQ53DpE0ddx216kWz6bT9lr+HRRASED/NBaWlmEMxswpzSPW5V23outWKQJraHszEcaVbwSeksAAA=="
+}
+
+```
+
+***
+### 7a: export signed psbt to environment variable
+
+:arrow_forward: `export SIGNED_PSBT="Paste_PSBT_HERE"`
+
+![](https://i.imgur.com/VJsl8zR.gif)
+
+
+
+
+
+***
+## Step 8: Broadcast Transaction 
+
+:arrow_forward:  `bdk-cli wallet --wallet wallet_name --descriptor $my_descriptor broadcast --psbt $SIGNED_ PSBT`
+
+
+![](https://i.imgur.com/yQZZk0d.gif)
+
+ 
+:+1: The output below confirms the command was successful.
+
+ ```
+{
+  "txid": "a0877b7ce91ea6d141ba63277673f5bdf0edfdd45f91a39ba1a1ace15f839b52"
+}
+ 
+
+ ```
+
+
+
+ - Verify transcation in the memory pool on testnet [Mempool-testnet!](https://mempool.space/testnet)
```

Comment:
Finishing touch, run sync one more time and see your balance decreased..

Reply:
Tip was added!